demos/pxp/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+cp /pxp-oneDNN-sample/build/examples/primitives-matmul-cpp ./
+occlum new occlum_workspace
```

Comment:
If and only if the occlum_workspace folder is not existing then run the occlum new command to create one.